tools/render-test/slang-support.cpp

```diff
@@ -149,30 +149,49 @@ static const char computeEntryPointName[] = "computeMain";
 
     if (request.computeShader.name)
     {
-        int computeEntryPoint = 0;
+        int computeEntryPointIndex = 0;
         if(!gOptions.dontAddDefaultEntryPoints)
         {
-            computeEntryPoint = spAddEntryPointEx(slangRequest, computeTranslationUnit,
+            computeEntryPointIndex = spAddEntryPointEx(slangRequest, computeTranslationUnit,
                 computeEntryPointName,
                 SLANG_STAGE_COMPUTE,
                 (int)rawEntryPointTypeNames.getCount(),
                 rawEntryPointTypeNames.getBuffer());
 
-            setEntryPointExistentialTypeArgs(computeEntryPoint);
+            setEntryPointExistentialTypeArgs(computeEntryPointIndex);
         }
 
         spSetLineDirectiveMode(slangRequest, SLANG_LINE_DIRECTIVE_MODE_NONE);
+
         const SlangResult res = spCompile(slangRequest);
+
         if (auto diagnostics = spGetDiagnosticOutput(slangRequest))
         {
             fprintf(stderr, "%s", diagnostics);
         }
 
         SLANG_RETURN_ON_FAIL(res);
 
+        // We are going to get the entry point count... lets check what we have
+        {
+            auto reflection = spGetReflection(slangRequest);
+            // Get the amount of entry points in reflection
+            const int entryPointCount = int(spReflection_getEntryPointCount(reflection));
+
+            // Above code assumes there is an entry point
+            SLANG_ASSERT(entryPointCount && computeEntryPointIndex < entryPointCount);
+
+            auto entryPoint = spReflection_getEntryPointByIndex(reflection, computeEntryPointIndex);
```

Comment:
This is a good cleanup to have, since it means that in the library case our compute tests don't have to care what the original entry point name had been.